templates/pipeline_execution/sdk_delete_pre_build_request.go.tpl

```diff
@@ -0,0 +1,12 @@
+	latestStatus := r.ko.Status.PipelineExecutionStatus
+	if latestStatus != nil {
+		if *latestStatus == svcsdk.PipelineExecutionStatusStopping {
+			return r, requeueWaitWhileDeleting
+		}
+
+		// Call StopPipelineExecution only if the job is Executing, otherwise just 
+		// return nil to mark the resource Unmanaged
```

Comment:
Is this comment out of date?

Reply:
Believe it is still valid, in the case where we want to delete a pipeline_execution and its status is Succeeded